src/core/ColorPicker/ColorPicker.tsx

```diff
@@ -58,20 +38,16 @@ export type ColorBuilderProps = {
 };
 
 export type ColorPaletteProps = {
-  /**
-   * Available color values to show in palette
-   * */
-  colors?: Color[];
-  /**
-   * Title shown above color palette
-   */
+  /** Available color values to show in palette */
+  colors?: ColorValue[];
```

Comment:
Disagree... one type of Color in / one type of Color out

Reply:
So its same type in, same out. In is the `SelectedColor`, out is `onChangeCompleted`.
Color list is a part of `ColorPicker` and that should not be tied to our input/output type.

Reply:
If you are talking about the color Palette colors, then in our use case, those colors are stored in a database as an array of tbgr values, I will convert this array of values to ColorValues so the conversion is done once. I don't want to have to convert to some other format that you like pass it in to the list just to have to convert them back to ColorValues internally.

Reply:
> In every current imodeljs use case we would want to pass the current color is via ColorValue

> I need to support an array of ColorValues.

I don't understand. Converting ColorValue to Color type is trivial, and makes the API more user friendly.

The user should not ever have to type `ColorValue` if they are not already using it imo

Reply:
To me it makes sense to use ColorValue since the onChange will give you a ColorValue back. It is the single source of truth for the color.  The old Color type have multiple different specifications in it.... it was NOT a single source of truth.  A user could specify {rgb: {r:255 g:255: b:255, colorString: 'rgb(0,0,0)'} hsl: {h:180, s:50, l:50, colorString: '#ocffdd'} where is the single source of truth there and which of the bogus values should be used?